### ISSUE TITLE
[Suggestion] Add a Cosmetic Armor mod

### ISSUE THREAD

**Issue by Balandala** [DenisMasterHerobrine/ElerionBloodNSteel#54]
Curseforge: https://www.curseforge.com/minecraft/mc-mods/cosmetic-armor-fabric
Modrinth: https://modrinth.com/mod/cosmetic-armor

Добавить мод на слоты для косметической брони, что бы можно было более гибко настраивать свой внешний вид, не жертвуя функциональностью.

**Comment by Gwinbleidddd** [comment]
Согласен, большинство брони, особенно на магов слишком специфичная (броня из ботании жестко ярких цветов, из спектрума подходит только под определённые скины, а из soulslike weaponry и more wizards броня откровенно страшная)

**Comment by LieSain1** [comment]
Есть что то в этом
